powsybl-parent/powsybl-parent-ws/pom.xml

```
@@ -54,29 +54,8 @@
         <git-id.failOnUnableToExtractRepoInfo>false</git-id.failOnUnableToExtractRepoInfo>
 
         <!-- for liquibase plugin -->
-        <liquibase-core.version>4.20.0</liquibase-core.version>
-        <!--
-          liquibase-hibernate6 4.20 (the same as liquibase-core) generates useless changesets, 4.23.2 doesn't seem to have this problem.
-          for example, but maybe there are more
-            existing
-            <column name="mystr" type="VARCHAR(255)">
-            still generates
-            <modifyDataType columnName="mystr" newDataType="varchar(255)"/>
-
-            existing
-            <column name="mydate" type="TIMESTAMP">
-            still generates
-            <modifyDataType columnName="mydate" newDataType="timestamp(6)"/>
-
-            existing
-            <column name="myuuid" type="UUID">
-            still generates
-            <modifyDataType columnName="myuuid" newDataType="bytea"/>
-
-            Switching to liquibase-hibernate.version 4.26.0, which includes resolving the issue where it attempted to connect to the database during changelog generation.
-            This connection issue has been corrected in version 4.26.0, along with potential additional bug fixes and enhancements.
-        -->
-        <liquibase-hibernate.version>4.26.0</liquibase-hibernate.version>
+        <liquibase-maven-plugin.version>4.29.2</liquibase-maven-plugin.version>
+        <liquibase-hibernate6.version>4.26.0</liquibase-hibernate6.version>
```

```suggestion
        <liquibase-hibernate6.version>4.29.2</liquibase-hibernate6.version>
```
Use same version as maven plugin ?